libjob_queue/tests/job_status_test.c

```diff
@@ -27,7 +27,7 @@
 
 void call_get_status( void * arg ) {
   job_queue_status_type * job_status = job_queue_status_safe_cast( arg );
-  job_queue_status_get_count( job_status , JOB_QUEUE_DONE + JOB_QUEUE_DO_KILL);
+  job_queue_status_get_count( job_status , 2*JOB_QUEUE_DO_KILL_NODE_FAILURE);  // This enum value is completly missing; should give util_abort.
```

Comment:
what about `pow(2, JOB_QUEUE_MAX_STATE)` instead of `2*JOB_QUEUE_...`? So we don't have to change the test in case we add more states

Reply:
Good idea - will update.